### PR TITLE
Implement functionality for Chapter 1, Homework 5

### PR DESCRIPTION
**Pull request by kungfuphil**
Add functionality to ArrayOperations methods.

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -9,7 +11,12 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+
+            Array.Sort(array);
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Please implement those manually.

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -19,7 +26,12 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+
+            Array.Reverse(array);
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Please implement those manually.

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -29,8 +41,20 @@ public static void Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:74]: Reuse removeAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -39,8 +63,20 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:88]: Reuse removeAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 5]
```diff
@@ -63,8 +113,21 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:125]: Reuse InsertAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 6]
```diff
@@ -75,8 +138,21 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:136]: Reuse InsertAt

**Review by Almantask — CHANGES_REQUESTED** [review]

**Comment by kungfuphil** [comment]
Can't believe I didn't think of reusing those methods! I've made all the recommended changes.

**Review by Almantask — APPROVED** [review]
Looks good 😄

**Pull request closed without merging**